lib/Cleantalk/ApbctWP/PluginSettingsPage/SummaryAndSupportRenderer.php

```diff
@@ -0,0 +1,428 @@
+            <div id="apbct_summary_and_support-sides_wrap">
+                <div id="apbct_summary_and_support-left_side">
+                    <h3 class="apbct_summary_and_support-side_header">%s</h3>
+                        <h4 class="apbct_summary_and_support-inner_header">%s</h4>
+                        %s
+                        <h4 class="apbct_summary_and_support-inner_header">%s</h4>
+                        %s
+                        <h4 class="apbct_summary_and_support-inner_header">%s</h4>
+                        %s
+                </div>
+                <div id="apbct_summary_and_support-right_side">
+                    <h3 class="apbct_summary_and_support-side_header">%s</h3>
+                    %s
+                </div>
+            </div>
+        </div>        
+        ';
+        $rendered = sprintf(
+            $template,
+            __('Summary'),
+            __('Statistics'),
+            $this->renderStatistics(), // Statistics will be rendered here
+            __('Sever requirements check'),
+            $this->renderServerRequirements(), // Server requirements will be rendered here
+            __('Connections'),
+            $this->renderConnectionReports(), // Connection reports will be rendered here
+            __('Support'),
+            $this->renderSupportActions()  // Support buttons will be rendered here
+        );
+        return $rendered;
+    }
+
+    /**
+     * Renders the statistics section containing various plugin performance metrics.
+     *
+     * Includes last request time, average response time, last SFW block, SFW update status,
+     * SFW logs, and plugin version information.
+     *
+     * @return string HTML content of the statistics section.
+     */
+    private function renderStatistics()
+    {
+        $render = '
+            <div class="apbct_summary-list_of_items">
+            %s
+            %s
+            %s
+            %s
+            %s
+            %s
+            </div>
+        ';
+
+        $render = sprintf(
+            $render,
+            $this->renderLastRequest(),
+            $this->renderAverageTime(),
+            $this->renderLastSfwBlock(),
+            $this->renderSfwUpdate(),
+            $this->renderSfwLogs(),
+            $this->renderPluginVersion()
+        );
+
+        return $render;
+    }
+
+    /**
+     * Renders the support buttons section with links to support resources.
+     *
+     * Includes buttons for opening support tickets, accessing help documentation,
+     * and creating a support user account with result display area.
+     *
+     * @return string HTML content of the support buttons section.
+     */
+    private function renderSupportActions()
+    {
+        $template = '
+        <div class="apbct_summary_and_support-support_buttons_wrapper">
+            <button type="button" class="cleantalk_link cleantalk_link-auto" onclick="apbctStatisticsOpenExternalLink(\'%s\')">
```

Comment:
Can we used the `<a>` tag insteaf of `<button>` for the preventing define and use extra JS-function `apbctStatisticsOpenExternalLink`?

Reply:
Done, thanks.